icu4j/main/localespi/pom.xml

```
@@ -215,6 +222,33 @@
         <localespi-tests.locale-providers>CLDR,COMPAT,SPI</localespi-tests.locale-providers>
       </properties>
     </profile>
+    <profile>
+      <id>with_javadoc</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+                <configuration>
+                  <!-- All files in com/ibm/icu/impl folders are exluded from javadoc.
+                    So we need this for bring back one file, to match the previously published artifacts.
```

Wording is confusing, probably the result of edits between 2 different ways of saying something.

Is this what you were trying to say?
```suggestion
                    So we need to bring back this one file, to match the previously published artifacts.
```